layerfetcher/layers.go

```diff
@@ -216,9 +216,10 @@ func (l *Logic) layerHashBlocksReqReceiver(ctx context.Context, msg []byte) []by
 
 	blocks := l.layerDB.GetLayerHashBlocks(h)
 	vector, err := l.layerDB.GetLayerInputVector(h)
```

Comment:
I think @y0sher told me getting input vector here is redundant, @y0sher ?

Reply:
i will double check with @y0sher
i want to remove this separately. i actually prefer keeping it here and don't make an extra network RTT at the end of PollLayerBlocks()